src/blocks/scratch3_looks.js

```diff
@@ -6,9 +6,10 @@ const RenderedTarget = require('../sprites/rendered-target');
  * @typedef {object} BubbleState - the bubble state associated with a particular target.
  * @property {Boolean} onSpriteRight - tracks whether the bubble is right or left of the sprite.
  * @property {?int} drawableId - the ID of the associated bubble Drawable, null if none.
+ * @property {Boolean} drawableVisible - if drawable has been hidden by blank text.
+ *      See _renderBubble for explanation of this optimization.
```

Comment:
Based on the description here it sounds like `drawableVisible` being true means that the Drawable is hidden. Based on the name of the property and the logic in `_renderBubble` it looks like `drawableVisible` being true means that the Drawable is visible. Unless I'm misunderstanding something, could you adjust for consistency? (Maybe just `false if drawable has been hidden by blank text.`?)

Reply:
Oh good catch, sorry I went back and forth adjusting these property names about a dozen times, must have got messed up in the shuffle.